### DISCUSSION TITLE
AI detection

### DISCUSSION THREAD

**Discussion by pz-max** [pypsa-meets-earth/pypsa-earth#13] [General]
# Some sources on AI detection:

- [Detecting HV power lines and substations](http://devseed.com/ml-grid-detection/). The [methodology](http://devseed.com/ml-grid-docs/) tells what DevSeed were doing in the [GitHub repo](https://github.com/developmentseed/ml-hv-grid-pub).
- [Detecting gas networks](https://elib.dlr.de/137734/1/thesis_jan_dasenbrock.pdf)
- Detecting [solar plants](https://dl.acm.org/doi/abs/10.1145/3396851.3397681)
- Detecting wind plants - [Deep wind](https://www.climatechange.ai/papers/neurips2019/5/paper.pdf)

**Comment by Cesare-Caputo** [comment]
Thanks for sending this over. @pz-max I thought power lines and substations was something you were already looking into? Or you want to enhance/ verify your results from what you have done already?  Been very busy recently but going to give this a go over the next 2 weeks and will keep you updated.

**Reply by pz-max** [reply]
We checked the methodology out as well as their GitHub repo. Just added the repo now. They performed the AI detection only for Pakistan, Kenya and Nigeria. I would suggest 1) we apply whatever they have and make sure that it is running and in step 2) we extend the approach to other countries.

**Reply by mnm-matin** [reply]
@pz-max where did you add the GitHub repo for that?

**Reply by pz-max** [reply]
@mnm-matin Link to their Github repo is above. Just forked it now. See pypsa-meets-africa repo
